kubernetes/pro/ingress/traefik-ingress-api.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```

Comment:
Sorry, I just noticed this, you are using `Ingress ` for pro and demo, but for standard version, `IngressRoute` is used. Is this an error or is there a reason for doing this differently?

Reply:
So we still need to install traefik to execute the ingress files for pro, right?

Reply:
Yes, if you want to those ingress manifests.  You could also create your own ingress with nginx, or HAProxy. And of course, you can still use port forwarding if you don't want to set up ingress,

Reply:
Understood, thank you